py-client/client_api.py

```diff
@@ -102,8 +102,7 @@ def create_session(self, image_in, expiry=0):
         fields = {}
         files = {'image': image_in}
 
-        status, response = AIAAUtils.http_multipart('PUT', self._server_url, selector, fields, files,
-                                                    multipart_response=False)
+        status, response, files = AIAAUtils.http_multipart('PUT', self._server_url, selector, fields, files)
```

Comment:
if the return file is expected to be None and we are not using it,
should we do
status, response, _ = AIAAUtils.http_multipart('PUT', self._server_url, selector, fields, files)

Reply:
fixed